pkg/apis/eksctl.io/v1alpha5/validation_test.go

```diff
@@ -368,11 +368,6 @@ var _ = Describe("ClusterConfig validation", func() {
 			checkItDetectsError(SSHConfig)
 		})
 
-		It("It doesn't panic when SSH is not enabled", func() {
```

Comment:
Uhm, if you remove this test from here then I think it should be moved somewhere else

Reply:
We have some other fields in `NodeGroup` that can be nil, but we don't have explicit tests for them. I think we should have a test that checks for default values after calling `SetNodeGroupDefaults`.